src/localize/languages/et.json

```diff
@@ -28,6 +28,6 @@
   "misc": {
     "aqi": "AQI",
     "humidity": "niiskus",
-    "feels-like": "Feels like"
+    "feels-like": "Tundub nagu"
```

Comment:
Isn't it "tuulekülm" in Estonian?

Reply:
> Isn't it "tuulekülm" in Estonian?

It looks to me like both are used, but I am not a native speaker and don't know if one is more common or preferred.

My understanding is that: 

Tundub nagu = "it feels like"
tuulekülm = "wind chill"

This pull request was more to simply get it away from English.  If you know that tuulekülm is preferred, feel free to make a pull request (or if you aren't comfortable doing that, I can)!